src/citrine/resources/__init__.py

```diff
@@ -1,2 +0,0 @@
-"""Resources that allow for interaction with the Citrine platform."""
-from citrine.resources.project import ProjectCollection  # noqa: F401
```

Comment:
No reason to play package magic with ProjectCollection.  Formally this is an API reduction, but seems accidental.